### ISSUE TITLE
New Sprites

### ISSUE THREAD

**Issue by MicroChasm** [nsmryan/RustRoguelike#199]
Sprite to be used for monsters that have been heard from behind a wall, but are unknown.

**Comment by MicroChasm** [comment]
Add four/five tiles to represent traps/trigger plates.

**Comment by MicroChasm** [comment]
Add cursor tile.

**Comment by MicroChasm** [comment]
New trap tiles added in the second to last row, on the first five columns (224-228). New unseen monster tile added in the last column of the second to last row (239).

**Comment by nsmryan** [comment]
I added these to the game, and pushed a new version to master.
They look good to me. I tried to choose traps that seemed to match their effect- the freeze trap looks the most like a snowflake, the gate trigger like a pressure plate, and the rightmost for the sound trap since it looks like it might make a sound.

One thing about the cursor- I think it might be a little dark or something. I feel like it should stand out, but no matter what color i give it, it ends up being a little muted.
